webcompat/templates/layout.html

```diff
@@ -27,11 +27,7 @@
   <link rel="icon" type="image/png" href="{{ url_for('static', filename='favicon/favicon-196x196.png') }}" sizes="196x196">
   <meta name="msapplication-TileColor" content="#e4bc05">
   <meta name="msapplication-TileImage" content="{{ url_for('static', filename='favicon/mstile-144x144.png') }}">
-{%- if config.PRODUCTION or config.STAGING -%}
-  <link href="{{ url_for('static', filename='css/webcompat.min.css')|bust_cache }}" type="text/css" rel="stylesheet">
-{%- else %}
-  <link href="{{ url_for('static', filename='css/webcompat.dev.css') }}" type="text/css" rel="stylesheet">
-{%- endif %}
+  <link href="https://webcompat.github.io/design/css/main.css" type="text/css" rel="stylesheet">
```

Comment:
Could you also add following? So we have it all done and the header works? :)

`<link href="https://fonts.googleapis.com/css?family=Open+Sans:400,600|Source+Sans+Pro:300,400|PT+Mono" rel="stylesheet">`

Reply:
Sure, done! Added a little `TODO` reminder as well, just to make sure we don't forget! :)